bodybank/character_generator/views.py

```diff
@@ -23,3 +27,35 @@ def get_ego(request):
 
 def say_thanks(request):
     return render(request, 'thanks.html')
+
+class OwnerMixin():
```

Comment:
It's a good exercise for Django devs to at least once use class based views. They save code, but if you need to do something a bit more complicated it becomes confusing soon!

Contrast this with the relative simplicity of function based views.

I buy into much of what Luke Plant says here:
https://spookylukey.github.io/django-views-the-right-way/

I wrapped the most important points into a preso:
https://docs.google.com/presentation/d/1GBp6xJtxcQi2r2MHc1Mn4a8Bkxkuzt9pmckY1LAzhio/edit#slide=id.g11fa00ee09f_0_10





[7:22](https://pybitesdevelo-ddf2687.slack.com/archives/D03S0PS3067/p1663824179665649)

If you prefer video you can watch the Code Clinic as well: https://pybites.mykajabi.com/products/pybites-coaching-calls-2/categories/3469727/posts/2156867625